plugin/storage/es/esRollover.py

```diff
@@ -19,6 +19,7 @@
 SHARDS = 5
 REPLICAS = 1
 
+TIMEOUT=120
```

Comment:
Please add a "help" doc entry like this: https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/es/esCleaner.py#L16

While you're at it, can you please fix [esCleaner's help doc for TIMEOUT](https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/es/esCleaner.py#L16) to include the default timeout? i.e. 
```
        print('TIMEOUT ...  number of seconds to wait for master node response (default {}).'.format(TIMEOUT))
```